ccloud/README.md

```diff
@@ -20,15 +20,16 @@ This repo has a few resources to help you validate your solutions on Confluent C
 It steps through the following workflow.
 
 * Log in to Confluent Cloud
-* Create a demo environment and cluster, and specify them as the default
-* Create create a user key/secret
-* Create a Service Account and API key and secret
+* Create a new environment and specify it as the default
+* Create a new Kafka cluster and specify it as the default
+* Create a user key/secret pair
```

Comment:
Like the env, and cluster, do the creds get "selected" also?

For instance, in my log specify active API key that was just created
ccloud api-key use ...

Reply:
Yes!  Thanks